### PR TITLE
Add bazel build support for `examples`.

### PR DESCRIPTION
**Pull request by HailongWen**
To resolve the transitive dependency, I used [migration-tooling](https://github.com/bazelbuild/migration-tooling) to auto-generate libraries definitions. Command can be found in the generated file, which might help when updating the build file in the future.

This PR is a sub-item of #855 and depends on #890 to be merged. I'll update the `README.md` later.

### CODE REVIEW

**Comment by bogdandrutu** [comment]
How do we make sure that we don't break this?

**Comment by HailongWen** [comment]
As @sebright suggested downloading bazel while building might take a while. Maybe we should add another build task to verify it?

**Comment by sebright** [comment]
I would rather create a new build job, since it allows more parallelism, and it might simplify the Travis script.  Do you think we should also build the examples on Windows and OS X with each build tool?

**Comment by bogdandrutu** [comment]
@sebright I am fine only on linux.

**Comment by HailongWen** [comment]
@bogdandrutu Of course. Done.

File: examples/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,61 @@
+load("//:opencensus_workspace.bzl", "opencensus_java_libraries")
+opencensus_java_libraries()
+
+java_library(
+    name = "examples",
```
Comment (sebright) [inline examples/BUILD.bazel]: opencensus-examples?

**Pull request merged**